langchain/src/llms/openai.ts

```diff
@@ -235,7 +237,11 @@ export class OpenAI extends BaseLLM implements OpenAIInput {
    * const response = await openai.generate(["Tell me a joke."]);
    * ```
    */
-  async _generate(prompts: string[], stop?: string[]): Promise<LLMResult> {
+  async _generate(
+    prompts: string[],
+    stop?: string[],
+    callbackManager?: CallbackManager
```

Comment:
this creates an issue with this PR https://github.com/hwchase17/langchainjs/pull/47